### ISSUE TITLE
Long running jobs

### ISSUE THREAD

**Issue by joshbenaron** [geofmureithi/apalis#145]
How does Apalis deal with long-running jobs (e.g. 2 hours)?

Does it tell the queue it's alive with a ping to Redis every n seconds?

**Comment by geofmureithi** [comment]
Yes, there is a background job future that checks for `Orphaned` jobs eg a job that is in running status but the worker has not pinged redis in 5min. You can configure this with `with_storage_config`
